messages/protobuf/pb/messages.proto

```diff
@@ -18,7 +18,7 @@
 syntax = "proto3";
 
 // Package pb defines MinBFT protocol messages in Protobuf format.
```

Comment:
This comment also should be updated: "Package *pb* defines ..."

Reply:
I guess we'd better omit the package name from the comment since the generated Go code still declares the package name as `pb`.